mobile/src/screens/chat/components/chat-link/chat-link.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from '@react-navigation/native';
+import React from 'react';
+
+import PlusIcon from '#assets/img/icons/plus.svg';
+import { Text, View } from '#libs/components/components';
+import { RootScreenName } from '#libs/enums/enums';
+
+import { styles } from './styles';
+
+const ChatLink: React.FC = () => {
```

Comment:
Remove this component, we should use our already created link component if you need to change something in it e.g. color, add property like color or style (like solid | outlined | secondary)

Reply:
Done. 
- I've added an icon and custom styles for a link.
- I use the VIew inside UILink for styling purposes. I am not sure this is a correct way.